shared/@hedvig-ui/TextArea/text-area.tsx

```diff
@@ -17,20 +22,37 @@ export const TextArea: React.FC<{
   placeholder: string
   value: string | undefined
   onChange: (value: string) => void
+  focus?: boolean
 } & Omit<TextAreaProps, 'onChange'>> = ({
   placeholder,
   value: inputValue,
   onChange,
+  focus,
   ...props
 }) => {
+  const textAreaRef = useRef<HTMLElement>(null)
+
+  useEffect(() => {
+    const setFocus = async () => {
+      await sleep(1)
+      if (textAreaRef.current && focus) {
+        textAreaRef.current.focus()
+      }
+    }
+
+    setFocus()
```

Comment:
Perhaps something like this?
```suggestion
    setTimeout( () => {
      if (textAreaRef.current && focus) {
        textAreaRef.current.focus()
      }
    }, 1000)
```

Reply:
But also - why do we wait one second?

Reply:
Because a line break is triggered